chall01/jnovotny.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3
+import sys
+
+def usage_exit():
+	print("usage: ./jnovotny.py <a-zA-Z string>")
+	exit()
+
+def morse(string):
+	morse_dict = {
+		"a":".-",
+		"b":"-...",
+		"c":"-.-.", 
+		"d":"-..",
+		"e":".",
+		"f":"..-.", 
+		"g":"--.",
+		"h":"....",
+		"i":"..",
+		"j":".---",
+		"k":"-.-",
+		"l":".-..",
+		"m":"--",
+		"n":"-.",
+		"o":"---",
+		"p":".--.",
+		"q":"--.-",
+		"r":".-.",
+		"s":"...",
+		"t":"-",
+		"u":"..-",
+		"v":"...-",
+		"w":".--",
+		"x":"-..-",
+		"y":"-.--",
+		"z":"--.."}
```

Comment:
in python, I would recommend putting this dict outside of the function, so if you use many times morse you don't declare this again.